borsh/tests/test_zero_size.rs

```diff
@@ -1,11 +1,35 @@
+use borsh::to_vec;
 use borsh::BorshDeserialize;
-
-#[derive(BorshDeserialize, PartialEq, Debug)]
-struct A;
+use borsh::BorshSerialize;
+#[derive(BorshDeserialize, BorshSerialize, PartialEq, Debug)]
```

Comment:
Let's add an empty line here

```suggestion

#[derive(BorshDeserialize, BorshSerialize, PartialEq, Debug)]
```